.github/workflows/ci.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   clippy:
     name: Clippy
     runs-on: ubuntu-latest
-    #    env:
-    #      RUSTFLAGS: -Dwarnings
+    env:
+        RUSTFLAGS: -Awarnings
```

Comment:
Explain: suppressing warnings from the GitHub actions helps one to find out which is the error, given that there are so many warnings.

Reply:
Sounds great!